apis_configs/fence_settings.py

```diff
@@ -29,6 +29,7 @@
 
 
 HOSTNAME = '{{hostname}}'
+HOST_NAME = '{{hostname}}'
```

Comment:
Lets just fix fence, because future us will wonder what we were thinking otherwise.